drivers/cst816s/Kconfig

```
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_CST816S
```

As there are no inner options we should have only `config` here, otherwise it shows as a menu.
```suggestion
config MODULE_CST816S
```